libgalois/include/katana/gstl.h

```diff
@@ -61,14 +62,19 @@ using List = std::list<T, FixedSizeAlloc<T>>;
 template <typename T, typename C = std::less<T>>
 using Set = std::set<T, C, FixedSizeAlloc<T>>;
 
+template <
+    typename T, typename Hash = std::hash<T>,
+    typename KeyEqual = std::equal_to<T>>
+using UnorderedSet = std::unordered_set<T, Hash, KeyEqual, Pow2Alloc<T>>;
+
 template <typename K, typename V, typename C = std::less<K>>
 using Map = std::map<K, V, C, FixedSizeAlloc<std::pair<const K, V>>>;
 
 template <
     typename K, typename V, typename Hash = std::hash<K>,
     typename KeyEqual = std::equal_to<K>>
-using UnorderedMap = std::unordered_map<
-    K, V, Hash, KeyEqual, FixedSizeAlloc<std::pair<const K, V>>>;
+using UnorderedMap =
+    std::unordered_map<K, V, Hash, KeyEqual, Pow2Alloc<std::pair<const K, V>>>;
```

Comment:
Fix `Map` and `Set` as well to use `Pow2Alloc`.

Reply:
I'm actually not sure if that works or not from what I remember; unordered vs ordered
containers seem to have different allocator requirements.

I'll test it out and see what happens I suppose.

Reply:
Thanks. If it doesn't work, please add a comment saying so.

Reply:
Seems to work from the bit of testing I did. New commit has it changed.